Set-SslSecurity.ps1

```diff
@@ -271,7 +297,7 @@ process {
 # Touch NOTHING below this line
 `$User = 'chocouser'
 `$SecurePassword = `$NexusUserPW | ConvertTo-SecureString -AsPlainText -Force
-`$RepositoryUrl = "https://`$(`$fqdn):8443/repository/ChocolateyInternal/index.json"
+`$RepositoryUrl = "https://`$(`$fqdn):8443/repository/ChocolateyCore/index.json"
```

Comment:
Do we need to keep _a_ definition of the Internal repo? We're still using it as an admin only source, right?

Reply:
Need both as one is base artifacts and other is default output from Jenkins jobs.